### PR TITLE
First file rewritten to TS

### PR DESCRIPTION
**Pull request by baierjak**
based on #34 

+ added jest types + renamed test file to TS

### CODE REVIEW

File: src/utilities/strictObjectAccess/strictObjectAccess.ts [hunk 1]
```diff
@@ -14,15 +14,15 @@ export class StrictAccessError extends Error {
...
         const HMR_PROPERTY_NAME = '$$typeof';
 
-        if (!Reflect.has(target, propKey) && propKey !== HMR_PROPERTY_NAME) {
-            throw new StrictAccessError(target, propKey);
+        if (!Reflect.has(target, propertyKey) && propertyKey !== HMR_PROPERTY_NAME) {
+            throw new StrictAccessError(target, propertyKey);
         }
 
-        return Reflect.get(...args);
+        return Reflect.get(target, propertyKey, receiver);
```
Comment (horaklukas) [inline src/utilities/strictObjectAccess/strictObjectAccess.ts:25]: If you changed it to not using `args` why don't just change the header to 

```ts
get(target. object, propertyKey: string, receiver) {
    const HMR_PROPERTY_NAME = '$$typeof';
     ...
}
```

File: src/utilities/strictObjectAccess/strictObjectAccess.ts [hunk 2]
```diff
@@ -33,8 +33,8 @@ const handler = {
     @return {Object}
 */
 export default function strictObjectAccess(
-    target = {},
-    enabled = process ? process.env.NODE_ENV === 'development' : true,
+    target: any = {},
+    enabled: any = process ? process.env.NODE_ENV === 'development' : true,
```
[the remark this answers is not in the document]
Comment (baierjak) [reply src/utilities/strictObjectAccess/strictObjectAccess.ts:37]: yess

File: src/utilities/strictObjectAccess/strictObjectAccess.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export class StrictAccessError extends Error {
-    constructor(target, propKey) {
+    constructor(target: Object, propertyKey: String) {
```
[the remark this answers is not in the document]
Comment (baierjak) [reply src/utilities/strictObjectAccess/strictObjectAccess.ts:2]: yess

File: src/utilities/strictObjectAccess/strictObjectAccess.ts (hunk 1 above)
Comment (baierjak) [reply src/utilities/strictObjectAccess/strictObjectAccess.ts:25]: thats right, I changed only lines where were TS errors :) 
but it definitaly makes sense :)

**Pull request merged**